python/tank/util/json.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2018 Shotgun Software Inc.
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+Utility methods for unserializing JSON documents.
+"""
+
+# We need to add this to the file or the import json will reimport this
+# module instead of importing the global json module.
+from __future__ import absolute_import
+
+import json
+
+
+def _convert_unicode_keys_to_string(input_value):
+    """
+    Converts any :class:`unicode` instances in the input value into a utf-8
+    encoded :class`str` instance.
+
+    :param input_value: Value to convert. Can be a scalar, list or dictionary.
+
+    :returns: A value with utf-8 encoded :class:`str` instances.
+    """
+
+    if isinstance(input_value, unicode):
+        return input_value.encode("utf8")
+
+    if isinstance(input_value, list):
+        return [_convert_unicode_keys_to_string(item) for item in input_value]
+
+    if isinstance(input_value, dict):
+        return dict(
+            (_convert_unicode_keys_to_string(k), _convert_unicode_keys_to_string(v))
+            for k, v in input_value.iteritems()
+        )
+
+    return input_value
+
+
+def load(
```

Comment:
I've reused the name of the methods in the Python module. Should I have gone for a more verbose name instead, like
load_from_file and load_from_string?